app/components/Edit/CategoriesDropdown.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import Select from 'react-select';
+import * as dataService from '../../utils/DataService';
+
+class CategoriesDropdown extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            selectedValue: '',
```

Comment:
Should this be renamed to `selectedValues` and initialized to `[]`? Looks like when you update it, it gets set to an array of values.